src/test/java/seedu/address/logic/commands/PermaSortCommandTest.java

```diff
@@ -0,0 +1,27 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Contains unit tests for PermaSortCommand
+ */
+public class PermaSortCommandTest {
```

Comment:
Would be nice to include test cases that check against the permanently saved file to see if using the command changes it.

Reply:
will delete this PR now and push again after permaSort is merged